routes/api_endpoints.js

```diff
@@ -0,0 +1,73 @@
+import express from 'express';
+
+const router = express.Router();
+const entries = [
+  { id: 0, title: 'Title 0', text: 'Text 0' },
+  { id: 1, title: 'Title 1', text: 'Text 1' },
+  { id: 2, title: 'Title 2', text: 'Text 2' }
+];
+
+
+// Default page with endpoints
+router.get('/', (req, res) => {
+  res.render(__dirname + '/howto.mustache');
+});
+
+
+// Create entries
+router.post('/api/v1/entries', (req, res) => {
+  const title = req.body.title;
+  const text = req.body.text;
+  const id = entries.length;
+
+  const entry = {
+    id,
+    title,
+    text,
+  };
+
+  res.send(
+    {
+      message: 'An entry has been created',
+      details: entry,
+    },
+  );
+  res.status(200).end();
```

Comment:
update the status code for post route to create entry

Reply:
Updated